jackrabbit-parent/pom.xml

```diff
@@ -64,8 +64,10 @@
     <slf4j.api.version>1.7.36</slf4j.api.version>
     <slf4j.version>1.7.36</slf4j.version> <!-- sync with logback version -->
     <logback.version>1.2.11</logback.version>
-    <java.version>1.8</java.version>
-    <java.version.signature>java18</java.version.signature>
+    <java.version>11</java.version>
```

Comment:
I would recommend not using this variable any longer as it hides the same named Java System Property: https://books.sonatype.com/mvnref-book/reference/resource-filtering-sect-properties.html#resource-filtering-sect-system-properties (also exposed via Maven)
Also usually dot notation is used for Maven plugin parameter properties. What about `javaBytecodeVersion`?

Reply:
Whatever; this was copied from Oak's pom. Consistency rules.

Can you fix this in Oak (you're the expert :-)) I'll then adjust.

Reply:
https://github.com/apache/jackrabbit-oak/pull/1271